src/main/java/org/solid/testharness/http/Client.java

```diff
@@ -140,6 +147,9 @@ public String getUser() {
         } else {
             builder.method(method, HttpRequest.BodyPublishers.noBody());
         }
+        if (version != null) {
+            builder.version(HttpClient.Version.valueOf(version));
+        }
```

Comment:
OK, I suppose this may answer my question in https://github.com/solid/specification-tests/pull/69 ?